bin/build.js

```diff
@@ -107,4 +107,4 @@ fs.readdirSync(iconsDir).forEach((file) => {
   n++;
 });
 
-console.log(`\nSuccess! ${n} icons was created.`);
+console.log(`\nSuccess! ${n} icons where created.`);
```

Comment:
```suggestion
console.log(`\nSuccess! ${n} icons were created.`);
```

Reply:
Man, that's embarrassing...😰 Correcting spelling and being wrong about it... I'll fix it asap!